src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 200) {
+    this.health = health;
+    this.name = 'Animal';
+  }
+
+  dead(healthPoints) {
```

Comment:
[METHOD NAME] The method name 'dead' is misleading. It should be named to reflect the action it performs, such as 'applyDamage', since it's modifying health based on health points.